Makefile

```diff
@@ -117,25 +107,31 @@ exec:
 exec0:
 	docker-compose exec php ash
 
-stop:
-	@echo "Removing containers for $(COMPOSE_PROJECT_NAME)"
-ifeq ($(shell docker inspect --format="{{ .State.Running }}" $(COMPOSE_PROJECT_NAME)_php 2> /dev/null),true)
-	docker-compose down
-endif
+up: net
+	@echo "Updating containers..."
+	docker-compose pull --parallel
+	@echo "Build and run containers..."
+	docker-compose up -d --remove-orphans
 
-clean: info stop
-	if [ -d "web/core" ]; then docker run --rm -v $(shell pwd):/mnt skilldlabs/$(IMAGE_PHP) ash -c "rm -rf /mnt/web/core"; fi
-	if [ -d "web/libraries" ]; then docker run --rm -v $(shell pwd):/mnt skilldlabs/$(IMAGE_PHP) ash -c "rm -rf /mnt/web/libraries"; fi
-	if [ -d "web/modules/contrib" ]; then docker run --rm -v $(shell pwd):/mnt skilldlabs/$(IMAGE_PHP) ash -c "rm -rf /mnt/web/modules/contrib"; fi
-	if [ -d "web/profiles/contrib" ]; then docker run --rm -v $(shell pwd):/mnt skilldlabs/$(IMAGE_PHP) ash -c "rm -rf /mnt/web/profiles/contrib"; fi
-	if [ -d "web/sites" ]; then docker run --rm -v $(shell pwd):/mnt skilldlabs/$(IMAGE_PHP) ash -c "rm -rf /mnt/web/sites"; fi
-	if [ -d "web/themes/contrib" ]; then docker run --rm -v $(shell pwd):/mnt skilldlabs/$(IMAGE_PHP) ash -c "rm -rf /mnt/web/themes/contrib"; fi
+down:
+	@echo "Removing network & containers for $(COMPOSE_PROJECT_NAME)"
+	@docker-compose down --remove-orphans
+
+clean: DIRS := core libraries modules/contrib profiles/contrib sites themes/contrib
+clean: info down
+	@for i in $(DIRS); do if [ -d "web/$$i" ]; then echo "Removing web/$$i..."; docker run --rm -v $(shell pwd):/mnt $(IMAGE_PHP) sh -c "rm -rf /mnt/web/$$i"; fi; done
 
 net:
-ifeq ($(strip $(shell docker network ls | grep $(COMPOSE_PROJECT_NAME))),)
-	docker network create $(COMPOSE_PROJECT_NAME)_front
+ifeq ($(shell docker network ls -q -f Name=$(COMPOSE_NET_NAME)),)
+	docker network create $(COMPOSE_NET_NAME)
+endif
+ifeq ($(shell grep -c -F 'IPRANGE=' .env), 0)
+#	@echo Define IP range $(net-range)
+	@printf "\nIPRANGE=%s\n" $(shell docker network inspect $(COMPOSE_NET_NAME) --format '{{(index .IPAM.Config 0).Subnet}}') >> .env
```

Comment:
It still executed before the recipe

Steps to reproduce `make down && make net`
```
$ make net
Error: No such network: sdd_front
Error: No such network: sdd_front
docker network create sdd_front
``